File: 3515-find-if-digit-game-can-be-won/3515-find-if-digit-game-can-be-won.cpp
# include<cmath>
class Solution {
public:
    bool canAliceWin(vector<int>& nums) {
        
        int r=0,s=0;
        for(int i=0;i<nums.size();i++){
            if(floor(nums[i]/10)){
                r+=nums[i];
            }
            else{
                s+=nums[i];
            }
        }
        if(s>r){
            return true;
        }
        else if(s<r){
            return true;
        }
        else{
        return false;}
        return false;
    }
};

File: 3515-find-if-digit-game-can-be-won/3515-find-if-digit-game-can-be-won.py
class Solution:
    def canAliceWin(self, nums: List[int]) -> bool:
        r=0
        s=0
        for i in range(len(nums)):
            
            if(nums[i]//10>=1):
                r+=nums[i]
            else:
                s+=nums[i]
        if(s>r):
            return True
        elif(s<r):
            return True
        else:
            return False
        return False
        